### PR TITLE
Add definition fujifilmprinter

### PR DESCRIPTION
**Pull request by Wooboy**
Please give a short description what your pull request is for

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [x] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.
- [x] If my Pull Request makes discovery/polling/yaml changes, I have added/updated [test data](https://docs.librenms.org/Developing/os/Test-Units/).

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: includes/definitions/fujifilmprinter.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+poller_modules:
+    printer-supplies:   true
+    arp-table: false
+    bgp-peers: false
+    vlans: false
+    ucd-diskio: false
+    stp: false
+    mpls: false
+    ucd-mib: false
+    ospf: false
```
Comment (Jellyfrog) [inline includes/definitions/fujifilmprinter.yaml:29]: dont disable modules if they dont cause issues.

File: includes/definitions/discovery/fujifilmprinter.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+modules:
+    os:
+        sysDescr_regex: '/FUJIFILM (?<hardware>[^;]+)/'
+        hardware: .1.3.6.1.4.1.297.1.11.93.2.1
```
Comment (Jellyfrog) [inline includes/definitions/discovery/fujifilmprinter.yaml:4]: this line seems wrong?

Comment (Wooboy) [reply includes/definitions/discovery/fujifilmprinter.yaml:4]: Sorry, I had a misunderstanding about the yaml, I've fixed it.

**Comment by Wooboy** [comment]
@Jellyfrog , The problematic files have been fixed, please help to review the contents.

File: includes/definitions/fujifilmprinter.yaml (hunk 1 above)
Comment (murrant) [reply includes/definitions/fujifilmprinter.yaml]: Most poller modules do a very quick check to see if they should poll based on the discovery module.  So, mostly it is enough to only disable the discovery module ...  and the discovery module runs only once every 6 hours so it should  not be very disruptive.

**Review by murrant — APPROVED** [review]

**Pull request merged**